openadapt/strategies/vanilla.py

```diff
@@ -195,7 +195,7 @@ def generate_action_event(
         current_window=current_window_dict,
         recorded_actions=recorded_action_dicts,
         replayed_actions=replayed_action_dicts,
-        replay_instructions=replay_instructions,
+        instructions=instructions,
```

Comment:
I believe this needs to stay as "replay_instructions", as per the .j2 file

Reply:
But replay fails.
If we use the replay command with `--instructions`, it fails with the error, undefined argument instructions.

We can do 2 things
either the command we use has`--replay_instructions` as argument OR vanilla.py changes replay_instructions -> instructions.

I would change it to whatever you might prefer

Reply:
To clarify:

this line should read `replay_instructions=instructions`

Reply:
Oh yess, it would, missed the line number. You are correct

Reply:
@abrichr updated the PR 🙏🏻